api/python/quilt3/main.py

```diff
@@ -41,7 +42,15 @@ def cmd_catalog():
         command += ["-e", var]
     command += ["-p", "3000:80", "quiltdata/catalog"]
     subprocess.Popen(command)
-    app.run()
+    print("Running a local version of the catalog at http://localhost:3000")
+
+    # app.run() generates misleading info to stderr ("Running on http://127.0.0.1:5000/") so disable stderr
```

Comment:
This seems like overkill. If we hide stderr, we'll hide exceptions, errors, etc. Let's add a line that clarifies it's the credential service running on 5000 then open the catalog URL so users can easily find the right URL.

Reply:
We can get rid of it if we launch the catalog automatically. Otherwise I couldn't find a great solution other than suppressing `stderr` or `stdout`. `app.run()` prints out a decent amount of noise so anything that we print gets lost in the noise while "Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)" is the last line printed.